packages/instantsearch.js/src/widgets/infinite-hits/__tests__/infinite-hits-integration-test.ts

```diff
@@ -486,9 +486,9 @@ describe('infiniteHits', () => {
 
       fireEvent.click(getByText(container, 'title 2'));
 
-      // The default `click` one + the custom one
-      expect(onEvent).toHaveBeenCalledTimes(2);
-      expect(onEvent.mock.calls[onEvent.mock.calls.length - 1][0]).toEqual({
+      // The custom one only
```

Comment:
Shouldn't it send the built-in `click` event as well? We're not dropping it when it's a `conversion` event.

Reply:
It's with `bindEvent`, so we don't know (easily) what kind of event was sent. Do you think it's needed to check inside the default event handler what kind of event is bound?

Reply:
Can we spend a bit more time to dig into how we can make it happen?

I understand we might be technically limited, but this differs from the scenario we agreed on in the [RAPID](https://docs.google.com/document/d/1HiPY7lK6sT6HNA9K0PP-El9yFSq-ZM74V6YCnlchH1k/edit#heading=h.jk50s25qbuc3). If we eventually realize it's not possible, it should be known in the team so it's not considered as a bug, and probably documented.

Reply:
In later commits I've made this work by moving bindEvent up the tree + sharing the "dedupe" logic between sendEvent + bindEvent calls :) This simplifies the logic significantly